cfg/std.cfg

```diff
@@ -7755,6 +7778,7 @@ initializer list (7) string& replace (const_iterator i1, const_iterator i2, init
   <podtype name="intmax_t,std::intmax_t" sign="s"/>
   <podtype name="uintmax_t,std::uintmax_t" sign="u"/>
   <podtype name="size_t" sign="u"/>
+  <podtype name="std::size_t" sign="u"/>
```

Comment:
@danmar Only `size_t` is configured, i am not sure why there is no `std::size_t` in std.cfg. I guess it does no harm if it is added. Otherwise, it is likely handled internally already.

Reply:
maybe. we somehow handle size_t internally.. its size is given by the platform configuration.

Reply:
There is this line in symboldatabase.cpp:
https://github.com/danmar/cppcheck/blob/master/lib/symboldatabase.cpp#L5707
So it could be unnecessary to specify `std::size_t` in std.cfg, but I guess it does not harm?
So I think this PR can be merged, what do you think?

Reply:
yes we can try it. feel free to merge.